content/installation/java/Java-linux-packages.md

```diff
@@ -0,0 +1,122 @@
+<!--
+title: "Java Agent Installation via Host Installer"
+description: "Overview of Java agent installation with Host Installer"
+tags: "installation Java agent linux Host Installer"
+-->
+
+## About Installation via System Package Managers
+
+The Contrast Java agent may be installed using system package managers. Unlike the *jar* file you can download from the Contrast UI, the Contrast Java agent *jar* installed by system package managers isn't preconfigured with connection parameters; instead, you must configure the Java agent using a [configuration file](installation-javaconfig.html).
+
```

Comment:
Should this 'configuration file' hyperlink wait until common config docs are finalized?

Reply:
yes